Geometry/TrackerGeometryBuilder/src/TrackerParametersFromDD.cc

```diff
@@ -0,0 +1,43 @@
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerParametersFromDD.h"
+#include "Geometry/TrackerNumberingBuilder/interface/GeometricDet.h"
+#include "CondFormats/GeometryObjects/interface/PTrackerParameters.h"
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "DetectorDescription/Core/interface/DDVectorGetter.h"
+#include "DetectorDescription/Base/interface/DDutils.h"
+
+bool
+TrackerParametersFromDD::build( const DDCompactView* cvp,
+				PTrackerParameters& ptp)
+{
+  std::vector<int> pxbPars = dbl_to_int( DDVectorGetter::get( "pxbPars" ));
+  putOne( GeometricDet::PixelBarrel, pxbPars, ptp );
```

Comment:
Better not to use GeometricDet::… enumerators here. Either use 1,2,…,6 or PixelSubdetector::PixelBarrel, PixelSubdetector::PixelEndcap, StripSubdetector::TIB, StripSubdetector::TID, StripSubdetector::TOB, StripSubdetector::TEC